### PR TITLE
Move CSS of adreport page into .css file

### PR DESCRIPTION
**Pull request by tomasko126**
Signed-off-by: Tomáš Taro tomas@getadblock.com

@kpeckett I’ve just moved inline CSS into single .css file.

Please take a look and review. Thanks :)

### CODE REVIEW

File: pages/adreport.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    margin-left: 18px;
+    vertical-align: top;
+}
+#step_disable_extensions_DIV {
+    position: relative;
+    margin: 0px auto;
+    max-width: 1000px;
+    padding: 20px 20px 20px 0px;
+}
+#step_report_DIV, #step_language_DIV, #step_update_filters_DIV, #step_firefox_DIV, #step_flash_DIV, #step_malware_DIV, #step_everywhere_DIV, #step_malware_checking_DIV, #step_response_success, #step_response_error  {
```
Comment (itskdog) [inline pages/adreport.css:84]: It's there any way that this line could become a class selector instead?

**Comment by tomasko126** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 7
           
```


See the complete overview on [Codacy](https://www.codacy.com/app/tomastaro/catblock/pullRequest?prid=241959&bid=3312099)

**Comment by tomasko126** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://www.codacy.com/app/tomastaro/catblock/pullRequest?prid=241959&bid=3312099)

**Comment by tomasko126** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://www.codacy.com/app/tomastaro/catblock/pullRequest?prid=241959&bid=3312099)

**Comment by tomasko126** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://www.codacy.com/app/tomastaro/catblock/pullRequest?prid=241959&bid=3312099)

**Pull request merged**